README.md

```diff
@@ -91,3 +91,8 @@ Questions & Comments
 --------------------
 
 For any and all feedback, please use the Issues on this repository. 
+
+License
+--------------------
+
+By downloading the .zip you are agreeing to the terms in the project [EULA](http://aka.ms/dotnet-cli-eula).
```

Comment:
I don't think the EULA for our rc2 release should point to the `master` branch, as that can change as we move forward with our development.

It probably shouldn't even point to rel/1.0.0 either because we will continue RTM development work in that branch, which could make changed to the EULA.

@piotrpMSFT - Are we going to be creating a "RC2" specific branch that this EULA should be pointing to?